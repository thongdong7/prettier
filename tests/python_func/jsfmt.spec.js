run_spec(__dirname, { parser: "python", tabWidth: 4 });
