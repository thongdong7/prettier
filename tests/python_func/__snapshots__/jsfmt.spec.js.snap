// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`args_default_func.py 1`] = `
def   abc(a , b , c, d=1,e=2,f=[]):

    pass~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
def abc(a, b, c, d=1, e=2, f=[]):
    pass
`;

exports[`args_func.py 1`] = `
def   abc(a , b , c):

    pass~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
def abc(a, b, c):
    pass
`;

exports[`args_kwargs_func.py 1`] = `
def   abc(a , b , c, **kwargs):

    pass~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
def abc(a, b, c, **kwargs):
    pass
`;

exports[`simplest_func.py 1`] = `
def   abc():

    pass~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
def abc():
    pass
`;
